project.clj

```diff
@@ -38,7 +38,11 @@
                   ;; infinite recursion, which I was not able to figure
                   ;; out.
                   :optimizations :simple
```

Comment:
did you try `:optimizations :advanced`? Update: bah, I tried it, and it causes an error:
```
Uncaught TypeError: ReactDOM.re is not a function
    at MB (compiled.js?t=1508518348002:734)
    at compiled.js?t=1508518348002:2445
    at compiled.js?t=1508518348002:2446
```